### ISSUE TITLE
Page scaling defaults

### ISSUE THREAD

**Issue by magd2884** [OpenBoard-org/OpenBoard#919]
### Describe the bug

The default aspect ratio has changed, and there is no option to set it back to what it was in prior versions.

### To Reproduce

1. Open OpenBoard 1.7.0 on Mac or Windows
2. Zoom out one click
3. Observe how the top and bottom of the page now appear, where before zooming out, they were clipped off.

#### Expected behavior

For OpenBoard to open at a Zoom level that matches the full "page" size of 1.33 : 1, or for this option to be available as a default.

![Page size (1 33 by 1)](https://github.com/OpenBoard-org/OpenBoard/assets/26061847/7918ee51-1aab-4401-b8e0-59798d601a7e)
(red markers shown for comparison to next image)

#### Actual behavior

OpenBoard opens with the top and bottom of the page clipped off.

![Visible size (1 43 by 1)](https://github.com/OpenBoard-org/OpenBoard/assets/26061847/959f21fe-bf11-437f-a479-fffe5e8b33ae)
(red markers shown for comparison to next image)

### Context

  - reproducibility : systematic
  - OpenBoard version :
      - bug happens with : 1.7.0 onwards
      - bug doesn't happen with : versions prior to 1.7.0
  - OS : 
      - bug happens with : Windows and macOS
      - didn't check : Linux
  - Installation source : OpenBoard download page
  
### Additional context

Our teachers are all using a system built with OpenBoard that will need rebuilding quite quickly if this is to be the new standard for OpenBoard.

So it'd be great to find out if this is how OpenBoard is going to be from now on, or if this is a temporary change.

The worst case scenario for us is if this is about to be fixed or changed back, and we rebuild everything now, and then have to rebuild everything again when it's changed back!

It'd be good to know what the plans are :)

Here's my MacOS, though this seems to be a problem for everyone...

<img width="450" alt="Mac OS" src="https://github.com/OpenBoard-org/OpenBoard/assets/26061847/bbfa1569-b27e-47c3-8df2-f9ab228a2e4c">

I'm using a second screen with resolution FHD 1080p

Thanks so much!

**Comment by letsfindaway** [comment]
Thanks for reporting. I found out that when calculating the initial zoom factor the height of the menu bar was not taken into account.

I just added a new commit fixing this as part of my PR #901 which is also addressing the scaling of the display view (second monitor).

**Comment by magd2884** [comment]
Great!

May I ask - if we're not coders, is this something that we'll have to wait for a new OpenBoard version for?

And if so, about how long will that take?

If it'll be more than a few weeks, we'll have to adapt our work in the meantime you see, so it'd be helpful to know.

Thanks again for all that you're doing!

**Comment by letsfindaway** [comment]
> Great!
> 
> May I ask - if we're not coders, is this something that we'll have to wait for a new OpenBoard version for?

Yes. I'm working on Linux and am not able to provide a MacOS version. So you will have to wait for a release where these changes are incorporated.
 
> And if so, about how long will that take?

Let me forward this question to @kaamui. I'm an OpenBoard contributor. My pull requests are just offerings for the project, but the project maintainer has to make the final decision to merge a pull request.
 
> If it'll be more than a few weeks, we'll have to adapt our work in the meantime you see, so it'd be helpful to know.
> 
> Thanks again for all that you're doing!

You're welcome :smile:
